### ISSUE TITLE
Does not take sampleRate input

### ISSUE THREAD

**Issue by yomed** [audiocogs/pitch.js#7]
In case anyone else runs into this: I noticed that the docs indicate `sampleRate` input as in `new PitchAnalyzer(sampleRate)`. But in actuality, it appears that the rate of 44100 is hardcoded, and the input is not used.

**Comment by apasov** [comment]
Yes, I can confirm that. In my case with 48000 sample rate pitch is not detected properly. I have to resample the audio buffer to 44100 before processing it with pitch.js

**Comment by mikemichaelis** [comment]
why not change the hard coded sample rate to 48000?  or change the prototype value before you new.
